DigitalLearningSolutions.Web/Scripts/searchSortAndPaginate/searchSortAndPaginate.ts

```diff
@@ -0,0 +1,100 @@
+import Details from 'nhsuk-frontend/packages/components/details/details';

```

Comment:
I'm assuming this is basically copy-paste of the old version - is there anything particular that I should focus on if it has been changed?

Reply:
Only major change other than the renaming is removing the GetPathFromEndpoint logic that was tying it to the /LearningPortal/ endpoints